demos-europe/demosplan-core
# F dplan 11470 create command to add permissions

### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-11470/3-0-BB-Als-Fachplanung-Administration-mochte-ich-keine-neuen-Verfahren-mehr-anlegen-konnen-um-falschliches-Anlegen-in-BLP-zu

Description:
1. We want a command that given customer id, role id, and permission, it enables this permission for all the orgas under the given customer and role.
2. The command has customerID roleID and permissionName parameters
3. The command also offers a dry-run, which does not execute the update, it only prints the impacted orgas
4. If customer or role or permission are not found, it throws an exception

### How to review/test
1. Typebin/blp dplan:permission:enable:customer-orga-role YOUR_CUSTOMER_ID YOUR_ROLE_ID CAN_CREATE_PROCEDURES --dry-run
2. Adjust param to insert ids based on your db
3. then check if the printed orgas are also inserted in the access_control DB

Delete the checkbox if it doesn't apply/isn't necessary.

- [x ] Tests updated/created
- [ x] Move the tickets on the board accordingly
